scripts/ci/build-and-test.sh

```diff
@@ -10,20 +10,29 @@ cd $(dirname $0)/../..
 source scripts/ci/sources/mode.sh
 source scripts/ci/sources/tunnel.sh
 
+# Check if tests can be skipped
+skip_tests=true
+for filename in $(git diff --name-only $TRAVIS_BRANCH...HEAD); do
```

Comment:
@DevVersion ``HEAD`` would be PR merged into ``master`` when ``$TRAVIS_BRANCH`` is set to ``master``. Then the diff would be your PR commits. You can try it.

I already tested this PR. And just did another one: 
https://travis-ci.org/chouclee/material2/builds/234685761 (``md`` file change only)
https://travis-ci.org/chouclee/material2/builds/234687221 (``md`` and ``js`` file, this is a hard push to update PR).

You can find the tests were skipped in first CI build, but not in second one.